### PR TITLE
FlipView remove caret scale on press/hover

### PR DESCRIPTION
**Pull request by beervoley**
## Description

This PR removes next/previous button caret scale on hover/press in FlipView
## Motivation and Context

Because the rest size of the caret is even (8px) while desired hover/pressed are 9px and 7px, it's very hard to properly center the icon. Design made a decision to not do scale animation.
## How Has This Been Tested?

Manually.

### CODE REVIEW

**Comment by beervoley** [comment]
@ranjeshj these resources:
image
were only added to be used for this scale animation and it was only a part of 2.6 prerelease. I think we should remove them because they're not being used anymore. Can we do that?

File: dev/CommonStyles/FlipView_themeresources.xaml [hunk 1]
```diff
@@ -85,8 +85,6 @@
     </ResourceDictionary.ThemeDictionaries>
 
     <x:Double x:Key="FlipViewButtonFontSize">8</x:Double>
-    <x:Double x:Key="FlipViewButtonFontSizePointerOver">9</x:Double>
-    <x:Double x:Key="FlipViewButtonFontSizePressed">7</x:Double>
```
Comment (beervoley) [inline dev/CommonStyles/FlipView_themeresources.xaml:89]: @ranjeshj @StephenLPeters want to make sure that we can remove these resources

Comment (StephenLPeters) [reply dev/CommonStyles/FlipView_themeresources.xaml:89]: This is my only question with the change as well.

**Review by StephenLPeters — APPROVED** [review]
:shipit:

File: dev/CommonStyles/FlipView_themeresources.xaml (hunk 1 above)
Comment (StephenLPeters) [reply dev/CommonStyles/FlipView_themeresources.xaml:89]: Ranjesh says its fine :)

**Comment by beervoley** [comment]
not merging until decision about caret scale is made

**Review by marksfoster — APPROVED** [review]

**Comment by Poopooracoocoo** [comment]
don't mind me: just as a user, scaling the size of an object on press/hover is hella annoying and i'm surprised _microsoft_ designers would do that

**Pull request closed without merging**